cdserver/lib/middleware/middleware_healthcheck.rb

```diff
@@ -0,0 +1,20 @@
+#
```

Comment:
This appears to be a duplicate of app/middleware/middleware_healthcheck.rb
Why do we need both?